packages/gatsby-plugin-sharp/src/__tests__/index.js

```diff
@@ -62,8 +62,8 @@ describe(`gatsby-plugin-sharp`, () => {
         file,
       })
 
-      expect(result.src.indexOf(file.name)).toBe(8)
-      expect(result.srcSet.indexOf(file.name)).toBe(8)
+      expect(result.src.indexOf(file.name)).toBe(19)
+      expect(result.srcSet.indexOf(file.name)).toBe(19)
```

Comment:
Those had to be changed because the strings changed from:
`/static/test-1234-7bb67.png`
to
`/static/1234/7bb67/test.png`
So now file name starts at index `19`.

Reply:
maybe we should adjust this test to test that final part of path is same as source file? i.e.
`expect(path.parse(result.src)).toBe(file.name)` ?

Reply:
and similiar with `srcSet` - probably would need some processing to split by `,` and then space or something like that